openslides_backend/action/actions/meeting/replace_projector_id.py

```diff
@@ -0,0 +1,39 @@
+from ....models.models import Meeting
+from ....shared.patterns import FullQualifiedId
+from ....shared.schema import required_id_schema
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData
+from .shared_meeting import meeting_projector_default_object_list
```

Comment:
rename to `meeting_projector_default_replacements`

Reply:
Done